source/testing-for-govready-q/automated-testing.rst

```diff
@@ -89,3 +91,72 @@ with serving the page.
 .. code-block:: bash
 
     python manage.py test -v 3
+
+Windows(WSL) users
+~~~~~~~~~~~~~~~~~~~~~
+
+Aside downloading (i.e. chromium-chromedriver) the executable to the system Windows needs a PATH that points to where the executable chromedriver file is located. This is also true for Windows Subsystem for Linux. A standard location to move the chromedriver executable to is `/usr/local/bin`. With that the program can just be named in your section of code without any other path parts needed.
+Below is an example of adding chromedriver as a parameter of the selenium chrome webdriver:
+
+.. code-block:: bash
+
+    driver = selenium.webdriver.Chrome(executable_path='chromedriver.exe')
+
+.. note::
+   The above applies for WSL 1 and for WSL2 there are a few more steps to properly use chromedriver. If you ever want to set your Ubuntu install to use a different WSL version then in a windows terminal run ``wsl --set-version Ubuntu <new_version_number>``
```

Comment:
@davidpofo Abbreviation spacing varies between "WSAL 1" vs "WSL2". Which is correct?

Reply:
ah thanks i'll change it to WSL 2